meshroom/ui/qml/main.qml

```diff
@@ -909,8 +909,8 @@ ApplicationWindow {
                     }
                     MaterialToolButton {
                         id: filePollerRefreshStatus
-                        text: _reconstruction.filePollerRefresh == 0 ? MaterialIcons.sync : MaterialIcons.sync_disabled
-                        ToolTip.text: "Set File Refresh Status"
+                        text: (_reconstruction.filePollerRefresh == 0 || _reconstruction.filePollerRefresh == 2) ? MaterialIcons.sync : MaterialIcons.sync_disabled
```

Comment:
Use enum values like PollerRefreshStatus.AUTO_ENABLED instead of 0.

Reply:
The enum is created on the Python side, and there is no way to expose it to QML with PySide2 (https://bugreports.qt.io/browse/PYSIDE-957).